.github/actions/setup-pip/action.yml

```diff
@@ -0,0 +1,29 @@
+# TODO(gha): Move this into `setup-sentry` action when
```

Comment:
Don't you mean, move `actions/cache` into here? And then, this isn't truly necessary, because then you mean you want `actions/cache` in `setup-sentry`. But I think it's still nice to have a separate setup-pip to dedupe.

Reply:
Yeah, I was thinking we'd want to move `actions/cache` and `setup-pip` both into `setup-sentry`, but you're right - it'd be better to move `actions/cache` here and have `setup-sentry` call `setup-pip` as necessary.